### PR TITLE
Feature/data merge

### PR DESCRIPTION
**Pull request by stevedoty**

### CODE REVIEW

File: src/Components/Team.js [hunk 1]
```diff
@@ -13,21 +14,21 @@ const Team = () => {
...
-                        <div className="card-body text-center">
-                          <p>
-                            <img className="img-fluid" src={person.imgUrl} alt="card image" />
-                          </p>
-                          <h4 className="card-title">{person.fullName}</h4>
-                          <p className="card-text">{person.title}</p>
-                        </div>
-                      </div>
+                      <Card>
+                        <Card.Body className="card-body text-center">
```
Comment (montoya332) [inline src/Components/Team.js]: Extra class card body may not be needed

Comment (stevedoty) [reply src/Components/Team.js]: just fixed it

File: src/Components/NavBar.js [hunk 2]
```diff
@@ -1,15 +1,16 @@
 import React from 'react';
+import Navbar from 'react-bootstrap/Navbar';
 
 const NavBar = () => {
   return (
-    <nav className="navbar navbar-expand-lg navbar-light fixed-top" id="mainNav">
+    <Navbar fixed="top" variant="light" id="mainNav">
       <div className="container">
-        <a className="navbar-brand js-scroll-trigger" href="#page-top">
+        <Navbar.Brand className="navbar-brand js-scroll-trigger" href="#page-top">
```
Comment (montoya332) [inline src/Components/NavBar.js]: Duplicate class

File: src/Components/Team.js [hunk 3]
```diff
@@ -13,21 +14,21 @@ const Team = () => {
...
-                        <div className="card-body text-center">
-                          <p>
-                            <img className="img-fluid" src={person.imgUrl} alt="card image" />
-                          </p>
-                          <h4 className="card-title">{person.fullName}</h4>
-                          <p className="card-text">{person.title}</p>
-                        </div>
-                      </div>
+
+                        <Card body className="card-body text-center">
```
Comment (montoya332) [inline src/Components/Team.js]: Nested cars body component needed . Remove body attr

File: src/Components/Team.js [hunk 4]
```diff
@@ -13,21 +14,21 @@ const Team = () => {
...
+                          <Card.Img className="img-fluid" src={person.imgUrl} alt="card image" />
```
Comment (montoya332) [inline src/Components/Team.js]: Not sure if image needs to be wrapped by span

File: src/index.css [hunk 5]
```diff
@@ -82,6 +82,9 @@ section h2 {
 }
 .helping-families-logo{fill:#1B1464;}
 
+/*!
+what is .navbar-toggler? can i delete this?
+ */
 #mainNav .navbar-toggler {
```
Comment (montoya332) [inline src/index.css]: Yea

**Pull request merged**